andela/Titan-Devs
# #164201481 Read, update, delete articles

#### What does this PR do?
- It helps the current user to read articles, update and delete their articles
#### Description of Task to be completed?
- The user should be able to read articles created by other users
- He also can update/delete the articles he authored
#### How should this be manually tested?
- After cloning the repository, check out `ft-wip-create-an-article-endpoints-164201481` branch
- Make sure to read `README.md` file to get the setup instructions
- Using any `API client tool(usually POSTMAN)`,
- Create the user and login to get the `access-token`
- Create some articles using `POST /api/v1/articles` endpoint
- Make sure to pass the token under `request headers`. It should be {`Authorization`: `Bearer <access-token>`} KEY, VALUE PAIR
- the request body should be as mentioned in the `README.md` file under `Endpoints:` ==> `Create Article` subtitle
After the successful article(s) creation,
- You can grab the slug of any of your articles
- Update that article using the `PUT /api/v1/articles/<slug>` 
- You also can delete it using `DELETE  /api/v1/articles/<slug>`
- You also can fetch all articles using `GET /api/v1/articles`

The `GET /api/v1/articles`  returns most recent articles globally.
#### Any background context you want to provide?
- None
#### What are the relevant pivotal tracker stories?
- #164201481
